.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: monthly
```

Maybe group them into one bundled PR, rather than having a separate PR for each action? No strong opinion, though

```suggestion
      interval: monthly
    groups:
      actions:
        patterns:
          - "*"
```

---

I wondered why we grouped them in python/typing, makes sense.